backend/migrations/0001_initial.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.3 on 2020-11-07 19:15
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Rule',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
Please, like the issue stated, can we make this a UUID instead of autoincrementing?

Reply:
Made it a UUID instead of auto increment.